components/View/Modal/index.js

```diff
@@ -56,7 +88,10 @@ export default class Modal extends React.PureComponent {
         return (
             <Portal>
                 <FocusTrap>
-                    <div className={this.getClassName()}>
+                    <div
+                        className={this.getClassName()}
+                        ref={this.onReference}
```

Comment:
To get a simple reference, we prefer to use the new method: ``` this.myRef = React.createRef(); ```

Reply:
@tnagorra changed the reference method